pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -256,6 +265,13 @@ func filterInstanceTypesByRequirements(instanceTypes []*cloudprovider.InstanceTy
 			results.remaining = append(results.remaining, it)
 		}
 	}
+	if requirements.HasMinValues() {
+		// We do not need minimum number of instanceTypes to satisfy all the minimum requirements here since we always send
```

Comment:
Should this comment talk about the 100 instance types or does it just need to say something like, "We don't care about the minimum number of instance types that meet our requirements here, we only care if they meet our requirements"

Reply:
Changed it.